doc/docs/master/deploy-manage/manage/s3gateway/index.md

```diff
@@ -25,8 +25,9 @@ not work in the browser. When you use authentication, set the access and
 secret key to the same value. They are both the Pachyderm auth token used
 to issue the relevant PFS calls.
 
-If auth is not enabled on the Pachyderm cluster, access credentials are
-ignored. You can not pass an authorization header, or if you do, any
+If auth is disabled, you can still pass arbitrary credentials, but the
+secret key must match the access key.
+You can not pass an authorization header, or if you do, any
 values for the access and secret keys are ignored.
```

Comment:
This sentence isn't right since access key and secret key must be the same. Also, it's a little confusing because someone might interpret it as "you must not pass an authorization header."

Reply:
removed